server/src/services/initialization/onInitialize.ts

```diff
@@ -79,7 +80,14 @@ export const onInitialize = (serverState: ServerState) => {
         codeActionProvider: true,
         hoverProvider: true,
         documentFormattingProvider: true,
-
+        semanticTokensProvider: {
+          legend: {
+            tokenTypes: tokensTypes,
+            tokenModifiers: [],
```

Comment:
What do you think is involved in adding modifiers in the future? Are there any technical blockers or is it just more time?

Reply:
I took a look at the available modifiers and decided to not include any at the moment (even though the code supports it), because all the useful ones would require name binding.